### PR TITLE
Make sure to catch tailwind classes from content

### PR DESCRIPTION
**Pull request by kevnk**
Make sure you [discussed your idea](https://github.com/studio1902/statamic-peak/discussions) before doing a PR.

Fixes # .

Changes proposed in this pull request:
-

### CODE REVIEW

**Comment by robdekort** [comment]
Hey thanks,

Not directly opposed to this, but wondering what the use case. For example I write blog articles with classes I don't want compiled.

**Comment by kevnk** [comment]
As I build pages for my clients, there's a lot of times when I find it easier and faster to just add HTML (and even antlers sometimes) directly to the page which is saved in the /content folder as a markdown file or yaml.

Other times I might have a drop-down to select a background color for a specific block on a page, and so I can use the class name as the value of the select option and confidently know that that class will be compiled on production.

Basically it is a way for me to ensure to have the right tailwind classes compiled when adding them through the control panel... At least during development when I can add that content, build locally, then push to production. When it's live, if the client adds their own tailwind classes, then it wouldn't work...

**Comment by robdekort** [comment]
Hmm I see. Please read this without judgment, but it's not how I do things. I can also see unexpected things happening with this approach. I feel like not merging this for now. But maybe someone else has any thoughts. We can always revisit.

Would you mind manually adding this to your Peak projects? Hope you understand.

**Comment by kevnk** [comment]
I totally understand! As I was explaining it I was thinking that it
might not be the best practice to have HTML in the control panel unless
you had some way of compiling tailwind in production.


On July 13, 2023, Giles Butler ***@***.***> wrote:
> Hmm I see. Please read this without judgment, but it's not how I do
> things. I can also see unexpected things happening with this approach.
> I feel like not merging this for now. But maybe someone else has any
> thoughts. We can always revisit.
>
> Would you mind manually adding this to your Peak projects? Hope you
> understand.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/studio1902/statamic-peak/pull/330#issuecomment-
1634660527>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAJAN3ARW3YSCT3XT5RWHDDXQAYVRANCNFSM6AAAAAA2JEOMOM>.
You are receiving this because you authored the thread.Message ID:
***@***.***>


--
Sent with HEY <https://hey.com/sent>

**Comment by robdekort** [comment]
Thanks. Happy to hear.

**Comment by marcorieser** [comment]
Just throwing my thoughts in here. This approach works fine as long as you build the css after every content change. This is, at least with my clients and the fact that they edit the content in production, no good working. A slightly more doable approach (but more restricted) is giving the client a list of whiteslisted classes which are baked in the tailwind config. Or adding them in the CP like the the Miniset addon does it in a way (https://github.com/jacksleight/statamic-miniset/blob/main/README.md#using-jit_safe-mode-with-tailwind).
